src/django_secret_sharing/utils.py

```diff
@@ -68,3 +76,10 @@ def validate_signed_id(signed_id, salt):
         return signing.loads(signed_id, salt=salt)
     except signing.BadSignature:
         return None
+
+
+def get_date_by_expires_value(expires_value):
+    try:
+        return timezone.now() + timedelta(seconds=int(expires_value))
```

Comment:
casten naar int hoeft niet omdat we al int hebben en zeker omdat int type required is (bovenstaande comment)